src/common/services/api/radioStations.service.spec.js

```diff
@@ -0,0 +1,30 @@
+import angular from 'angular'
+import 'angular-mocks'
+import module from './radioStations.service'
+
+describe('radio station service', () => {
+  beforeEach(angular.mock.module(module.name))
+  var self = {}
+
+  beforeEach(inject(function (radioStationService, $httpBackend) {
+    self.radioStationService = radioStationService
+    self.$httpBackend = $httpBackend
+  }))
+
+  afterEach(() => {
+    self.$httpBackend.verifyNoOutstandingExpectation()
+    self.$httpBackend.verifyNoOutstandingRequest()
+  })
+
+  it('getRadioStations', () => {
```

Comment:
This doesn't make sense. It should be `describe('getRadioStations')` with some `it(...)` calls underneath that make sense.